core/src/components/textarea/textarea.md.outline.scss

```diff
@@ -0,0 +1,221 @@
+ * a top and bottom border. This gives the appearance
+ * of a unified container with a border.
+ */
+:host(.textarea-fill-outline) .textarea-outline-start,
+:host(.textarea-fill-outline) .textarea-outline-notch,
+:host(.textarea-fill-outline) .textarea-outline-end {
+  border-top: var(--border-width) var(--border-style) var(--border-color);
+  border-bottom: var(--border-width) var(--border-style) var(--border-color);
+}
+
+/**
+ * Ensures long labels do not cause the notch to flow
+ * out of bounds.
+ */
+:host(.textarea-fill-outline) .textarea-outline-notch {
+  max-width: calc(100% - var(--padding-start) - var(--padding-end));
+}
+
+/**
+ * This element ensures that the notch used
+ * the size of the scaled text so that the
+ * border cut out is the correct width.
+ * The text in this element should not
+ * be interactive.
+ */
+:host(.textarea-fill-outline) .notch-spacer {
+  /**
+   * We need $textarea-md-floating-label-padding of padding on the right.
+   * However, we also subtracted $textarea-md-floating-label-padding from
+   * the width of .textarea-outline-start
+   * to create space, so we need to take
+   * that into consideration here.
+   */
+  @include padding(null, #{$textarea-md-floating-label-padding * 2}, null, null);
+
+  font-size: calc(1em * #{$textarea-floating-label-scale});
+
+  opacity: 0;
+  pointer-events: none;
+}
+
+:host(.textarea-fill-outline) .textarea-outline-start {
+  @include border-radius(var(--border-radius), 0px, 0px, var(--border-radius));
+  @include border(null, null, null, var(--border-width) var(--border-style) var(--border-color));
+
+  /**
+   * There should be spacing between the translated text
+   * and .textarea-outline-start. However, we can't add this
+   * spacing onto the notch because it would cause the
+   * label to look like it is not aligned with the
+   * text textarea. Instead, we subtract a few pixels from
+   * this element.
+   */
+  width: calc(var(--padding-start) - #{$textarea-md-floating-label-padding});
+}
+
+:host(.textarea-fill-outline) .textarea-outline-end {
+  @include border(null, var(--border-width) var(--border-style) var(--border-color), null, null);
+  @include border-radius(0px, var(--border-radius), var(--border-radius), 0px);
+
+  /**
+   * The ending outline fragment
+   * should take up the remaining free space.
+   */
+  flex-grow: 1;
+}
+
+/**
+ * When the textarea either has focus or a value,
+ * there should be a "cut out" at the top for
+ * the floating/stacked label. We simulate this "cut out"
+ * by removing the top border from the notch fragment.
+ */
+:host(.has-focus.textarea-fill-outline.textarea-label-placement-floating) .textarea-outline-notch,
+:host(.has-value.textarea-fill-outline.textarea-label-placement-floating) .textarea-outline-notch,
+:host(.textarea-fill-outline.textarea-label-placement-stacked) .textarea-outline-notch {
+  border-top: none;
+}
+
+:host(.textarea-fill-outline) .native-textarea {
+  @include border-radius(0);
```

Comment:
Is this still needed? The border radius was moved to `textarea-wrapper`

Reply:
Nope, confirmed it is no longer needed.